### PR TITLE
GPIB-422

### PR DESCRIPTION
**Pull request by xiangzhe0812**
fix duplicate button

### CODE REVIEW

File: src/utils/Verify.js [hunk 1]
```diff
@@ -4,12 +4,16 @@ import gpib from "../apis/gpib";
 
 const Verify = ({ setIdVerificationStatus, statuses, user }) => {
   useEffect(() => {
-    const script = document.createElement("script");
+    const scriptExist = document.getElementById("digitalIdScript")
+    let script;
```
Comment (ahinchliff) [inline src/utils/Verify.js]: don't need this `let` anymore

File: src/utils/Verify.js [hunk 2]
```diff
@@ -4,12 +4,16 @@ import gpib from "../apis/gpib";
...
+    if (scriptExist) {
+      scriptExist.remove();
+    }
 
-    script.src = `${process.env.REACT_APP_DIGITAL_SOURCE}`
-    script.async = true;
-
-    document.body.appendChild(script);
+    const divElement = document.getElementById("digitalid-verify");
+    console.log(divElement)
```
Comment (ahinchliff) [inline src/utils/Verify.js]: console.log(

Comment (bitcoinbrisbane) [inline src/utils/Verify.js]:
```suggestion
    console.log(divElement);
```

File: src/utils/Verify.js [hunk 3]
```diff
@@ -4,12 +4,16 @@ import gpib from "../apis/gpib";
...
+    script = document.createElement("script")
```
Comment (bitcoinbrisbane) [inline src/utils/Verify.js]:
```suggestion
    script = document.createElement("script");
```

File: src/utils/Verify.js [hunk 4]
```diff
@@ -4,12 +4,16 @@ import gpib from "../apis/gpib";
...
+    document.body.appendChild(script)
```
Comment (bitcoinbrisbane) [inline src/utils/Verify.js]:
```suggestion
    document.body.appendChild(script);
```

**Review by ahinchliff — CHANGES_REQUESTED** [review]

File: src/utils/Verify.js [hunk 5]
```diff
@@ -42,9 +46,14 @@ const Verify = ({ setIdVerificationStatus, statuses, user }) => {
         }
       });
     };
-  }, [setIdVerificationStatus, statuses, user]);
+    script.src = `${process.env.REACT_APP_DIGITAL_SOURCE}`
```
Comment (bitcoinbrisbane) [inline src/utils/Verify.js]:
```suggestion
    script.src = `${process.env.REACT_APP_DIGITAL_SOURCE}`;
```

**Review by ahinchliff — APPROVED** [review]

**Review by SharmaDinesh — APPROVED** [review]

**Pull request merged**